tests/integration_tests/scheduler/test_lsf_driver.py

```diff
@@ -52,6 +52,22 @@ async def test_lsf_stdout_file(tmp_path, job_name):
     assert "yay" in lsf_stdout
 
 
+async def test_lsf_dumps_stderr_to_file(tmp_path, job_name):
+    os.chdir(tmp_path)
+    driver = LsfDriver()
+    failure_message = "failURE"
```

Comment:
Is it on purpose or just typo: "failURE" ?

Reply:
On purpose. It crossed my mind that there is a above-zero probability that something else could write the string "failure" to the file, but exactly zero probability that something else would write "failURE"